core/src/test/java/org/infinispan/distexec/DistributedExecutorFailoverTest.java

```diff
@@ -3,10 +3,8 @@
 import java.io.Serializable;
 import java.util.List;
 import java.util.Random;
-import java.util.concurrent.Callable;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.Future;
-import java.util.concurrent.TimeUnit;
+import java.util.Scanner;
+import java.util.concurrent.*;
```

Comment:
Could you please remove star imports? Also, please make sure you're using the latest formatter: https://github.com/infinispan/infinispan/tree/master/ide-settings